oneflow/user/kernels/model_update_kernel_util.cpp

```diff
@@ -166,20 +173,27 @@ template struct AdamUpdateKernelUtil<DeviceType::kCPU, double, double>;
 template<typename T, typename K, typename IDX>
 struct IndexedSlicesAdamMdUpdateKernelUtil<DeviceType::kCPU, T, K, IDX> {
   static void Update(DeviceCtx* ctx, float beta1, float beta2, float epsilon, float weight_decay,
+                     bool amsgrad, bool do_bias_correction, float lr, int64_t train_step_val,
                      int64_t num_instance, int64_t feature_size, int64_t lower_bound,
                      int64_t upper_bound, const IDX* num_unique_instance,
-                     const float* learning_rate, const K* indices, const T* values, T* model, T* m,
-                     T* v) {
-    const float lr = *learning_rate;
+                     const float* learning_rate, const int64_t* train_step, const K* indices,
+                     const T* values, T* model, T* m, T* v, T* max_v) {
+    if (learning_rate != nullptr) { lr = *learning_rate; }
+    if (train_step != nullptr) { train_step_val = *train_step; }
+
+    train_step_val += 1;  // Step is start from 1.
+
     const int64_t n = *num_unique_instance * feature_size;
     FOR_RANGE(int64_t, i, 0, n) {
       const IDX indices_idx = i / feature_size;
       const IDX inner_idx = i - indices_idx * feature_size;
       const IDX instance_id = indices[indices_idx];
+
       if (instance_id >= lower_bound && instance_id < upper_bound) {
         const IDX model_idx = (instance_id - lower_bound) * feature_size + inner_idx;
-        AdamUpdateFunctor<T, T>()(values + i, model + model_idx, m + model_idx, v + model_idx, 1, 0,
-                                  0, beta1, beta2, epsilon, weight_decay, lr);
+        AdamUpdateFunctor<T, T>()(values + i, model + model_idx, m + model_idx, v + model_idx,
+                                  max_v + i, 1, 0, 0, beta1, beta2, epsilon, weight_decay, amsgrad,
+                                  do_bias_correction, lr, train_step_val);
```

Comment:
```suggestion
        AdamUpdateFunctor<T, T>()(values + i, model + model_idx, m + model_idx, v + model_idx,
                                  max_v + i, /*scale=*/1.0, /*l1=*/0.0, /*l2=*/0.0, beta1, beta2, epsilon, weight_decay, amsgrad,
                                  do_bias_correction, lr, train_step_val);
```

对于这种传参是一个数的时候，建议这么写，可以清楚的看出来哪个参数传的是什么值、什么类型的